blocks/map/visually-hidden.css

```diff
@@ -0,0 +1,12 @@
+.visually-hidden{
```

Comment:
The **BEM mix** `visually-hidden` is not an element of the map block and should be placed in its own directory within the block directory, named _visually-hidden_

Please, remember to update its import in `index.css`.

Reply:
@naumch1k I did it